<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькуляктор</string>
  </property>
  <widget class="QLineEdit" name="firstNumberEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondNumberEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="plusButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="minusButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="multiplyButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="divideButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstNumberLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Число 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondNumberLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Число 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Результат</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calcButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Посчитать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
